nx-libs.spec

```diff
@@ -1,4 +1,8 @@
 %global _hardened_build 1
+# prevent "ERROR: No build ID note found" errors
+%if 0%{rhel} < 7
+%undefine _missing_build_ids_terminate_build
```

Comment:
We have a different compiler in the path and I forgot to update the path prior to compiling. Mock would have helped here....